model/gen-ai/registry.yaml

```diff
@@ -117,6 +127,8 @@ groups:
         type: int
         brief: The number of tokens used in the GenAI response (completion).
         examples: [180]
+        note: >
+          If applicable. For example, embeddings operations to do use output tokens.
```

Comment:
Reviewer note: In the PR description I added an "Open question" about whether it would be good/helpful to add "If applicable" notes (or additions to the `brief`) for the span attributes that do not apply to embeddings operations. This is one such note to feel it out. Thoughts?

Reply:
Does this "to do" should be "do not"?

Reply:
"if applicable" usually goes on the `requirement_level" when referencing an attribute. No need to add it on the registry level - all attributes are used when they are appliable

Reply:
@lmolkova 

1. Do you mean like this?

https://github.com/open-telemetry/semantic-conventions/blob/5517079850f4a1bedfbb376354e2a2dfe74edc86/model/process/registry.yaml#L107-L118

That's the only example I've seen of `requirement_level` with both some explanatory text and a key other than `conditionally_required`.

See commit d79bf33.

2. Do you think there is value adding this kind of "if applicable for the particular GenAI operation" to most of the attributes? Or does that end up being too much table node spam?